applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -1423,7 +1423,7 @@ static bool event_handler(const struct event_header *eh)
 		return handle_wheel_event(cast_wheel_event(eh));
 	}
 
-	if (!IS_ENABLED(CONFIG_DESKTOP_BUTTONS_NONE) &&
+	if (IS_ENABLED(CONFIG_CAF_BUTTONS) &&
```

Comment:
`CONFIG_CAF_BUTTON_EVENTS`

note that we don't care what is the source of the event - only that something delivers it

Reply:
Similar as for `Kconfig.click_detector`.